napari/_qt/_qapp_model/qactions/_view.py

```diff
@@ -0,0 +1,91 @@
+from typing import List
+
+from app_model.types import Action, KeyCode, KeyMod, StandardKeyBinding
+
+from ...._app_model.constants import CommandId, MenuGroup, MenuId
+from ....settings import get_settings
+from ....utils.translations import trans
+from ...qt_main_window import Window
+from ...qt_viewer import QtViewer
+
+
+def _tooltip_visibility_toggle():
+    settings = get_settings().appearance
+    settings.layer_tooltip_visibility = not settings.layer_tooltip_visibility
+
+
+VIEW_ACTIONS: List[Action] = [
```

Comment:
Also, some tests will iterate over actions and check if the callback executes without exceptions?

Reply:
i agree that would be good, but will make a task issue for it.  That's essentially creating tests for a ton of potentially untested stuff.  100% useful, of course, but also well beyond the scope of this PR.